modules/crop-ffmpeg/src/main/java/org/opencastproject/crop/impl/CropServiceImpl.java

```diff
@@ -0,0 +1,403 @@
+  public static final String FFMEG_BINARY_DEFAULT = "ffmpeg";
+
+  /**
+   * The load intorduced on the system by creating a caption job
+   */
+  public static final float DEFAULT_CROP_JOB_LOAD = 1.0f;
+
+  /**
+   * The key to look for in the service configuration file to override the DEFAULT_CROP_JOB_LOAD
+   */
+  public static final String CROP_JOB_LOAD_KEY = "org.opencastproject.job.load.cropping";
+
+  /**
+   * The load introduced on the system by creating a cropping job
+   */
+  private float cropJobLoad = DEFAULT_CROP_JOB_LOAD;
+
+  /**
+   * The logging facility
+   */
+  protected static final Logger logger = LoggerFactory.getLogger(CropServiceImpl.class);
+
+  /**
+   * Reference to the receipt service
+   */
+  protected ServiceRegistry serviceRegistry = null;
+
+  /**
+   * The mpeg7 service
+   */
+  protected Mpeg7CatalogService mpeg7CatalogService = null;
+
+  /**
+   * The workspace to use when retrieving remote media files
+   */
+  protected Workspace workspace = null;
+
+  protected SecurityService securityService = null;
+
+  protected OrganizationDirectoryService organizationDirectoryService = null;
+
+  protected UserDirectoryService userDirectoryService = null;
+
+  /**
+   * Creates a new instance of the crop service.
+   */
+  public CropServiceImpl() {
+    super(JOB_TYPE);
+    this.binary = FFMEG_BINARY_DEFAULT;
+  }
+
+  @Override
+  public void activate(ComponentContext cc) {
+    super.activate(cc);
+    final String path = cc.getBundleContext().getProperty(FFMPEG_BINARY_CONFIG);
+    this.binary = StringUtils.defaultIfBlank(path, FFMEG_BINARY_DEFAULT);
+    logger.debug("Configuration {}: {}", FFMPEG_BINARY_CONFIG, FFMEG_BINARY_DEFAULT);
+  }
+
+  protected Track crop(Job job, Track track) throws CropException, MediaPackageException {
+
+    if (!track.hasVideo()) {
+      throw new CropException("Element is not a video track");
+    }
+    try {
+      Track croppedTrack;
+
+      File mediaFile;
+      try {
+        mediaFile = workspace.get(track.getURI());
+      } catch (NotFoundException e) {
+        throw new CropException("Error finding the video file in the workspace", e);
+      } catch (IOException e) {
+        throw new CropException("Error reading the video file in the workspace", e);
+      }
+
+      logger.info("starting cropping of {} ", track);
+
+      croppedTrack = cropFfmpeg(mediaFile, track);
+
+      Track cropTrack = (Track) MediaPackageElementBuilderFactory.newInstance().newElementBuilder()
+              .newElement(Track.TYPE, track.getFlavor());
+      URI uri;
+      try {
+        uri = croppedTrack.getURI();
+      } catch (Exception e) {
+        throw new CropException("Unable to put the mpeg7 catalog into the workspace", e);
+      }
+      cropTrack.setURI(uri);
+
+      logger.info("Finished video cropping of {}", track.getURI());
+
+      return cropTrack;
+
+    } catch (Exception e) {
```

Comment:
I think the only thing throwing an exception in here is 

      croppedTrack = cropFfmpeg(mediaFile, track);

You could just put a `try {} catch (IOException)` around that.